### PR TITLE
Update default for `HVACSizingControl/ManualJInputs/NumberofOccupants`

### LINKED ISSUE #1841: Update default for `HVACSizingControl/ManualJInputs/NumberofOccupants`
Manual J is not very clear on whether Nocc=Nbr + 1 is required or just the default. In multiple places, it says things like "The number of occupants shall equal the number of bedrooms, plus one." It makes it sound like it's basically required.

But I just found an ACCA document that says:
image

So it sounds like the OS-HPXML default should change from `Nbr+1` to `[Nbr+1, Nocc].max`.

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Fixes #1841.

## Checklist

Not all may apply:

- [x] ~Schematron validator (`EPvalidator.xml`) has been updated~
- [x] ~Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)~
- [x] Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests/test*.rb` and/or `workflow/tests/test*.rb`)
- [x] Documentation has been updated
- [x] ~Changelog has been updated~
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: workflow/tests/base_results/results_simulations_hvac.csv [hunk 1]
```diff
@@ -405,7 +405,7 @@ base-residents-0.xml,6.8,91.76,36000.0,24000.0,0.0,32239.0,8709.0,7508.0,0.0,575
...
 base-residents-1-misc-loads-large-uncommon2.xml,6.8,91.76,36000.0,24000.0,0.0,33431.0,8742.0,7508.0,0.0,575.0,6918.0,0.0,0.0,1738.0,2171.0,5779.0,0.0,0.0,21277.0,6150.0,7037.0,0.0,207.0,448.0,0.0,0.0,0.0,2293.0,622.0,0.0,4520.0,0.0,0.0,139.0,0.0,-661.0,0.0,800.0
 base-residents-1.xml,6.8,91.76,36000.0,24000.0,0.0,32239.0,8709.0,7508.0,0.0,575.0,6918.0,0.0,0.0,1738.0,2171.0,4620.0,0.0,0.0,20039.0,6112.0,7037.0,0.0,207.0,448.0,0.0,0.0,0.0,2293.0,622.0,0.0,3320.0,0.0,0.0,139.0,0.0,-661.0,0.0,800.0
-base-residents-5.xml,6.8,91.76,36000.0,24000.0,0.0,32173.0,4674.0,7508.0,0.0,575.0,6918.0,0.0,0.0,1738.0,2171.0,8590.0,0.0,0.0,16931.0,1046.0,7758.0,0.0,207.0,448.0,0.0,0.0,0.0,2293.0,1860.0,0.0,3320.0,0.0,0.0,-1175.0,0.0,-1975.0,0.0,800.0
+base-residents-5.xml,6.8,91.76,36000.0,24000.0,0.0,32173.0,4674.0,7508.0,0.0,575.0,6918.0,0.0,0.0,1738.0,2171.0,8590.0,0.0,0.0,17176.0,1061.0,7758.0,0.0,207.0,448.0,0.0,0.0,0.0,2293.0,1860.0,0.0,3550.0,0.0,0.0,-975.0,0.0,-1975.0,0.0,1000.0
```
Comment (joseph-robertson) [inline workflow/tests/base_results/results_simulations_hvac.csv:408]: Increases because `5 > 3+1`?

**Review by joseph-robertson — APPROVED** [review]
Worth a Changelog.md entry? If not, good to merge.

File: workflow/tests/base_results/results_simulations_hvac.csv (hunk 1 above)
Comment (shorowit) [reply workflow/tests/base_results/results_simulations_hvac.csv:408]: Yep.

**Comment by shorowit** [comment]
> Worth a Changelog.md entry? If not, good to merge.

I figured it can be part of [misc improvements](https://github.com/NREL/OpenStudio-HPXML/blob/845b6ab238e386fec9e82298f6a950ad904a4f3a/Changelog.md?plain=1#L26).

**Pull request merged**